scripts/gen_cri.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+set -euo pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+CONTAINERS_CRI="$SCRIPT_DIR/../granulate_utils/containers/cri/generated/"
+
+# released at Oct 26, 2018
+wget -O gogo.proto https://raw.githubusercontent.com/gogo/protobuf/v1.3.2/gogoproto/gogo.proto
+python3 -m grpc_tools.protoc -I. --python_out="$CONTAINERS_CRI" gogo.proto
+# released at Aug 27, 2021
+wget -O api.proto https://raw.githubusercontent.com/kubernetes/cri-api/v0.24.0-alpha.2/pkg/apis/runtime/v1alpha2/api.proto
```

Comment:
I saw online that it's the version being used. I suppose v1 is not supported yet everywhere (old containerd etc). Not sure how to check it.... hmm will think

Reply:
We decided to stay with `v1alpha2` since it now seems to work everywhere. Once new containerds / cri-os are deployed with no alpha support, we'll add support for v1 and try v1 then v1alpha2.